jax/random.py

```diff
@@ -44,6 +46,11 @@
 from jax.interpreters import xla
 from jax.util import prod
 
+FLAGS = flags.FLAGS
+flags.DEFINE_bool('jax_enable_masking_prng',
+                  bool_env('JAX_ENABLE_MASKING_PRNG', False),
```

Comment:
Can we rename this flag not to be about masking, perhaps just call it "prefix_prng"?